bhs_api/item.py

```diff
@@ -189,6 +189,15 @@ def enrich_item(item, db=None, collection_name=None):
     if pictures:
         main_image_id = None
         for image in pictures:
+            # Add 'PictureUrl' to all images of an item
+            image_id = image['PictureId']
```

Comment:
you are sure that image will always have 'PictureId'? if not, then it will raise an exception here